roles/openshift_health_checker/test/elasticsearch_test.py

```diff
@@ -50,10 +54,10 @@ def assert_error(error, expect_error):
 
 
 def test_check_elasticsearch():
-    assert 'No logging Elasticsearch pods' in canned_elasticsearch().check_elasticsearch([])
+    assert 'No logging Elasticsearch pods' in elasticsearch().check_elasticsearch([])
```

Comment:
When we make it a Pytest Fixture, we can use it by simply listing it as an argument to the test function: https://docs.pytest.org/en/latest/fixture.html#fixtures-as-function-arguments

But since it seems the intention is to call the function with different args, then perhaps it doesn't even need to be a fixture, but simply a plain function?